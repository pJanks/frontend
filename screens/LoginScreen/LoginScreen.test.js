import React from 'react';

describe('placeholder', () => {
  test('placeholder test', () => {
    expect(true).toEqual(true);
  })
})