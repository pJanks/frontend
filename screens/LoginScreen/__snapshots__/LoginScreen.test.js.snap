// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#b2e1f4",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "height": 250,
        "justifyContent": "center",
        "width": 250,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 100,
          "marginTop": 50,
          "width": 300,
        }
      }
    />
    <Image
      source={1}
      style={
        Object {
          "height": 250,
          "width": 250,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 18,
          "fontWeight": "bold",
          "textAlign": "left",
        }
      }
    >
      Username:
    </Text>
    <TextInput
      allowFontScaling={true}
      rejectResponderTermination={true}
      style={
        Object {
          "backgroundColor": "#fff",
          "height": 40,
          "paddingLeft": 10,
          "width": 250,
        }
      }
      underlineColorAndroid="transparent"
    />
    <Text
      style={
        Object {
          "fontSize": 18,
          "fontWeight": "bold",
          "textAlign": "left",
        }
      }
    >
      Password:
    </Text>
    <TextInput
      allowFontScaling={true}
      rejectResponderTermination={true}
      style={
        Object {
          "backgroundColor": "#fff",
          "height": 40,
          "paddingLeft": 10,
          "width": 250,
        }
      }
      underlineColorAndroid="transparent"
    />
    <View
      style={
        Object {
          "margin": 20,
          "width": 250,
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityStates={Array []}
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "padding": 8,
                  "textAlign": "center",
                },
              ]
            }
          >
            Login
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
